### PR TITLE
Update Copyright to Kokkos v. 3.0

### PR DESCRIPTION
**Pull request by ndellingwood**
Matches changes to master branch in PR #604

### CODE REVIEW

**Comment by ndellingwood** [comment]
Do I need to add a tag and push it with this PR as well?

File: Copyright.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@
 //               KokkosKernels: Linear Algebra and Graph Kernels
```
Comment (ndellingwood) [inline Copyright.txt]: Oop, this may need to be changed also

**Comment by ndellingwood** [comment]
Good catch by @vqd8a 
e.g. does this need to be NTESS? https://github.com/kokkos/kokkos-kernels/blob/a10bbc71310c3a07c1391060fe8de528a82df101/Copyright.txt#L25-L26

**Comment by crtrott** [comment]
Yeah it does need to change ...

**Comment by vqd8a** [comment]
Should it be `NTESS AND THE CONTRIBUTORS` in here?
https://github.com/kokkos/kokkos-kernels/blob/a10bbc71310c3a07c1391060fe8de528a82df101/Copyright.txt#L25

**Comment by vqd8a** [comment]
NTESS in here too, https://github.com/kokkos/kokkos-kernels/blob/a10bbc71310c3a07c1391060fe8de528a82df101/Copyright.txt#L28

**Comment by crtrott** [comment]
just NTESS

**Comment by ndellingwood** [comment]
Should be fixed now

**Comment by ndellingwood** [comment]
> Should it be NTESS?
> 
> https://github.com/kokkos/kokkos-kernels/blob/f049d325cadb5145e0b067bb3e614b262b22c721/Copyright.txt#L22

@vqd8a I think "Corporation" here is generically used - @crtrott can you verify?

**Comment by ndellingwood** [comment]
Merge conflicts resolved, could someone please review?

**Comment by srajama1** [comment]
Thanks @ndellingwood !

**Comment by crtrott** [comment]
There are still Kokkos v. 2.0 things in there.

**Comment by ndellingwood** [comment]
Pushed update fixing Kokkos v. 2.0 instances

**Pull request merged**